apps/docs/src/app/core/component-docs/list/list-header/list-header.component.html

```diff
@@ -1,10 +1,42 @@
 <header>List</header>
-<description
-    >The List Component can be used to display a list of items with simple information such as scopes, names, etc.
-    <br /><br />
-    List item anchors will be keyboard navigable via the tab key provided they have <code>[routerLink]</code> or
-    <code>[attr.href]</code> attributes. List action buttons and checkboxes can be navigated via the tab key. List
-    checkboxes can be selected with the space bar.
+<description>
+    In SAP Fiori, we distinguish between tables and lists.
+    Both usually contain homogeneous data, but lists generally have rather basic data,
+    whereas the data in tables tends to be more complex. Lists are mostly used in the master list
+    for a master-detail scenario using the flexible column layout, as well as in popovers or dialogs.
+
+    The list component displays a list of items with basic homogeneous information.
+
+    List item anchors, action buttons and checkboxes are keyboard navigable via the Tab key, provided that they have
+    <code>[routerLink]</code> or <code>[attr.href]</code> attributes.
+    List checkboxes can also be selected with the space bar.
+
+    <h3>Usage</h3>
+
+    <h4>Use the list if:</h4>
+    <ul>
+        <li>
+            You want to display a homogeneous set of basic data.
+        </li>
+        <li>
+            You need to sort, group, or filter simple datasets.
+        </li>
+        <li>
+            You need to display a single-level hierarchy for a simple use case.
+        </li>
+    </ul>
+
+    <h4>Do not use the list if:</h4>
+    <ul>
+        <li>
+            You want to manage complex datasets that need to be extensively sorted, grouped, filtered, or edited.
+            Instead, use <a routerLink="/core/table">Table Component</a>.
```

Comment:
Just link "Table". I would remove Component from that sentence entirely. Or say "Instead, use the [Table] component." I put brackets around where it should be linked. :)